docs/tutorials/thresholding.md

```diff
@@ -34,7 +34,7 @@ A *thresholder* in QuPath is something that can apply a threshold to an image wh
 
 ### Thresholds in action
 
-You can create a thresholder using the *Create thresholder* command.
+You can create a thresholder using the *Create thresholder* command within {menuselection}`Classify --> Pixel classification --> Create thresholder.`
```

Comment:
Suggest:
> You can create a thresholder using the {menuselection}`Classify --> Pixel classification --> Create thresholder`>  command.

Reason: we should probably use 'menuselection' consistently, since we can protect text in there from automatic translaters (with [a bit of extra effort](https://twitter.com/petebankhead/status/1614629700604674049?s=20&t=Xk9rzyctAigy2FdRWozWdQ)).